<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTMainWindow</class>
 <widget class="QMainWindow" name="PTMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuExecute"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSet_Mark"/>
   <addaction name="actionMove"/>
   <addaction name="actionZoom"/>
   <addaction name="actionFit_View"/>
   <addaction name="actionFlightDirection"/>
   <addaction name="actionFoto_Tri"/>
   <addaction name="actionView_Report"/>
  </widget>
  <action name="actionSet_Mark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/mark.png</normaloff>:/icon/mark.png</iconset>
   </property>
   <property name="text">
    <string>actionSet_Mark</string>
   </property>
   <property name="toolTip">
    <string>Marca o ponto na imagem</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/move.png</normaloff>:/icon/move.png</iconset>
   </property>
   <property name="text">
    <string>move</string>
   </property>
   <property name="toolTip">
    <string>Move a imagem</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/find.png</normaloff>:/icon/find.png</iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom na Imagem</string>
   </property>
  </action>
  <action name="actionFit_View">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fit.png</normaloff>:/icon/fit.png</iconset>
   </property>
   <property name="text">
    <string>fit</string>
   </property>
   <property name="toolTip">
    <string>Mostra toda a imagem</string>
   </property>
  </action>
  <action name="actionFoto_Tri">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/exterior.png</normaloff>:/icon/exterior.png</iconset>
   </property>
   <property name="text">
    <string>fotoTri</string>
   </property>
   <property name="toolTip">
    <string>Dispara o calculo da fotoTri</string>
   </property>
  </action>
  <action name="actionView_Report">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/table.png</normaloff>:/icon/table.png</iconset>
   </property>
   <property name="text">
    <string>viewReport</string>
   </property>
   <property name="toolTip">
    <string>Ve os resultados</string>
   </property>
  </action>
  <action name="actionFlightDirection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fd.png</normaloff>:/icon/fd.png</iconset>
   </property>
   <property name="text">
    <string>flightdirection</string>
   </property>
   <property name="toolTip">
    <string>Indica a direção de vôo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
